README.md

```diff
@@ -236,6 +236,33 @@ To do so you need to set `IDELOCALHOST` in you local.env file to the IP of your
 
 xdebug connections will then be sent to this IP address on port 9000.
 
+## Overriding / Extending
+
+You can add additional services, or modify current services, by creating a `docker-compose.override.yml` file. For example, to add a Redis service, add these contents to `docker-compose.override.yml`:
+
+```
+version: '2'
+services:
+  redis:
+    image: redis
+```
+
+To modify a current service, for example to specify volume caching for macOS:
+
+```
+version: '2'
+services:
+  web:
+  volumes:
```

Comment:
Does this merge or replace? Could link to relevant docs :-)

Reply:
Oops, I thought I had but I must have erased that. I'll add a link to https://docs.docker.com/compose/extends/